lua/orgmode/treesitter/headline.lua

```diff
@@ -475,6 +475,37 @@ function Headline:parse(pattern)
   return matching_nodes[1], match
 end
 
+function Headline:get_drawer(name)
+  local section = self.headline:parent()
+  local q = tree_utils.parse_query('(body (drawer) @drawer)')
```

Comment:
I think drawer is always in the first level of the headline, so you can probably just iterate the named children like we do when finding property drawer. Parsing queries is usually slow for the first time until it gets cached, so we try to avoid them where not necessary.

Reply:
Right now, it's below `body`, so one level down, but should not be difficult to find by iterating